File: Basic Java/TestingSystem_Assignment/src/com/vti/backend/TestingSystem_Assignment_4/CongNhan.java
package com.vti.backend.TestingSystem_Assignment_4;

public class CongNhan extends CanBo{
    private int level;

    public CongNhan() {
    }

    public CongNhan(long id, String fullName, int age, Gender gender, String address, int level) {
        super(id, fullName, age, gender, address);
        this.level = level;
    }
}

File: ReactJS/Assignment_02/Lab_02/Congnhan.ts
import {CanBo, Gender} from './CanBo';


export class Congnhan extends CanBo {
    private level: number;


	constructor(fullName: string, age: number, gender: Gender, address: string, level: number) {
        super(fullName, age, gender, address);
		this.level = level;
	}


    /**
     * Getter level
     * @return {number}
     */
	public getLevel(): number {
		return this.level;
	}

    /**
     * Setter level
     * @param {number} value
     */
	public setLevel(value: number) {
		this.level = value;
	}

    public displayDetails(){
        super.displayDetails();
        console.log( ` Level: ${this.getLevel()} `);
    }
    
}
